packages/contracts-bedrock/src/cannon/MIPS2.sol

```diff
@@ -728,9 +730,9 @@ contract MIPS2 is ISemver {
         uint256 s = 0;
         assembly {
             s := calldatasize()
-            innerThreadRoot_ := calldataload(add(THREAD_PROOF_OFFSET, 166))
+            innerThreadRoot_ := calldataload(add(THREAD_PROOF_OFFSET, 322))
         }
         // verify we have enough calldata
-        require(s >= (THREAD_PROOF_OFFSET + 198), "insufficient calldata for thread witness"); // 166 + 32
+        require(s >= (THREAD_PROOF_OFFSET + 322), "insufficient calldata for thread witness"); // 356 + 32
```

Comment:
Something looks not right here - either `322` is wrong, or the comment is wrong, or (I suspect) both.

Reply:
Ah good spot; Yeah, both are wrong.

```suggestion
        require(s >= (THREAD_PROOF_OFFSET + 354), "insufficient calldata for thread witness"); // 322 + 32
```